src/pages/guides/FAQ/FAQ -11-24

```diff
@@ -0,0 +1,58 @@
+    * V4 represents an expansion from primarily serving post-production video professionals to a discipline-agnostic platform for creatives and creative managers.  [Frame.io](http://frame.io/) acts as a central canvas for multi-disciplinary teams and solves for key needs for everything from more focus time for creative work to increased visibility into ongoing projects.
+    * **Workflow management**: Centralize, organize, and build all your end-to-end creative workflows at scale leveraging Metadata, Custom Fields, and Collections.
+    * **File transfer and management**: This set of capabilities gives users all the flexibility and power to build the best file management system for their creative process making it easier to quickly get the information they need, and manage all their work-in-progress assets at scale.
+    * **Review and approval**: [Frame.io](http://frame.io/) helps creative teams reduce the total number of revision cycles with advanced review, commenting, and annotation capabilities available across a broad set of media types.
+    * **Sharing and presentation**: Customize and share creative work from one central place.
+    * **Security and governance**: View and manage all users, workspaces, and projects in one place. Take bulk actions and export information to control how you utilize usage data.
+    * **Integrations and APIs**: Native integrations and an open API connect [Frame.io](http://frame.io/) to the systems you already use to unblock sticking points and keep work in flow.
+
+* How can I move my **Azure** **Blob** storage to [Frame.io](http://frame.io/) V4 directly?
+
+    The easiest and most efficient way to upload and download bulk content, large folder structures, and entire projects. Upload your files and folder structures directly from your desktop into your [Frame.io](http://frame.io/) projects; download the original files or choose multiple proxies. 
+
+* What is **metadata**?
+  
+  Use Metadata to tag, organize, and view creative assets based on how your team works. Tag all incoming assets with metadata to organize then group into Collections to sort and view your assets based on need (assignee, due date, approval status, etc.) for easier access and superior organization. 
+* What are **Shares**?
+   
+   Preview, edit, manage and send beautiful Shares to ensure your assets are showcased in the best light for internal and external stakeholders. Fine tune every element of your Share—from branding, to descriptions, to layout—to ensure that you are presenting your work professionally and beautifully.
+* What are **Collections** in V4?
+   
+   Collections are flexible, real-time, saved views of a selection of assets that allow users to dynamically select, filter, group, and sort their media using previously tagged metadata. Collections allow users to build meaningful, efficient workflows without the tool forcing them to work within a particular approach, process or template.
```

Comment:
I'm wondering if we're beyond the scope here of what I would expect would be frequently asked questions about the V4 API.

Reply:
Took out non-V4 API topics.